user_docs/en/userGuide.t2t

```diff
@@ -2865,6 +2874,13 @@ Specifically, following issues can be solved by running this tool:
 
 ++ Reload plugins ++[ReloadPlugins]
 This item, once activated, reloads app modules and global plugins without restarting NVDA, which can be useful for developers.
+You can also find out whether the application that you focus has an associated app module.
+
+%kc:beginInclude
+|| Name | Desktop key | Laptop key | Touch | Description |
+| Report loaded plugin and executable | ``NVDA+control+f1`` | ``NVDA+control+f1`` | None | Report the name of the app module associated to the current application, if any, and the name of the executable of the currently running application. |
+| Reload plugins | ``control+f3`` | ``NVDA+control+f3`` | None | Reloads NVDA's global plugins and app modules. |

```

Comment:
Could you put this row in first position in the table? That's the main subject of this paragraph.